app/plugin_install.go

```diff
@@ -148,10 +148,12 @@ func (a *App) removePlugin(id string) *model.AppError {
 
 	// Remove bundle from the file store.
 	storePluginFileName := filepath.Join("./plugins", manifest.Id) + ".tar.gz"
-	if bundleExist, _ := a.FileExists(storePluginFileName); bundleExist {
+	if bundleExist, fileErr := a.FileExists(storePluginFileName); bundleExist && fileErr == nil {
```

Comment:
it's more go-idiomatic if you check the error first and let the control fall through for success case, kinda like

```
bundleExist, fileErr := a.FileExists(storePluginFileName)
if fileErr != nil {
    return model.NewAppError("removePlugin", "app.plugin.remove_bundle.app_error", nil, err.Error(), http.StatusInternalServerError)
}

if bundleExist {
....
}
```

https://github.com/golang/go/wiki/CodeReviewComments#indent-error-flow